### DISCUSSION TITLE
Inspiration: Eagle app (media management, but also Electron)

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#149] [Random]
I would like to share another Electron app that I find very good: [Eagle](https://eagle.cool/).
It's about media file management, but here are a few things that I like about it that might be transferable to Nota as well:

- Speed: Despite handling lots of media, this app starts and acts very quickly
- Search: You have lots of filters (when was it changed? which tags does it have? etc)
- API: You can interact with your library via [API](https://api.eagle.cool/)

**Comment by astoilkov** [comment]
I didn't know about Eagle. I spend some time exploring it and I should say it's quite nice.

Regarding your points:
- Speed: We've made a few performance improvements to 0.34.0. Do you feel places or times when Nota is slow? Where should we improve?
- Search: We believe in search a lot. We are constantly making improvements to our algorithm as we have ideas for more search-related products. This is one of the main reasons we built [iBar](https://ibar.app). After we are ready with the core algorithm we will start experimenting with filters. Do you have any specific suggestions for the search in Nota?
- API: I've never seen such an approach. It's interesting. I will be looking at that app, its API, design, and features and we indeed can draw a lot of inspiration from it.

Thanks for the inspiration! We ourselves like some great apps and it's nice to hear about more. It seems like you have a good taste for apps. Any other recommendations?
